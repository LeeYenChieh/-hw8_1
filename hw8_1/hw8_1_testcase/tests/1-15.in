wAtzjweE9IKTMX57vV yZLgJ3rBwodNyOdsnKY2RInknpvY3agvivNZXu9O27ofZOyz6nNTKIl0sd3K1AP
wAtzjweE9IKTMX57vV chrJw1sZ37wdrUHrTxiizUdZIe84vq4IdV1URK9z6iAcaHRoT1P7r9zLt
wAtzjweE9IKTMX57vV display
wAtzjweE9IKTMX57vV display
wAtzjweE9IKTMX57vV display
wAtzjweE9IKTMX57vV display
wAtzjweE9IKTMX57vV add thickBlackBorder scrollBar
wAtzjweE9IKTMX57vV display
wAtzjweE9IKTMX57vV U3Q2S3tLGWkx
wAtzjweE9IKTMX57vV display
wAtzjweE9IKTMX57vV add scrollBar
wAtzjweE9IKTMX57vV display
wAtzjweE9IKTMX57vV display
wAtzjweE9IKTMX57vV display
wAtzjweE9IKTMX57vV add thickBlackBorder
