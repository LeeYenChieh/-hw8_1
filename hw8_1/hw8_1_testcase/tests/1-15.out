chrJw1sZ37wdrUHrTxiizUdZIe84vq4IdV1URK9z6iAcaHRoT1P7r9zLt 
chrJw1sZ37wdrUHrTxiizUdZIe84vq4IdV1URK9z6iAcaHRoT1P7r9zLt 
chrJw1sZ37wdrUHrTxiizUdZIe84vq4IdV1URK9z6iAcaHRoT1P7r9zLt 
chrJw1sZ37wdrUHrTxiizUdZIe84vq4IdV1URK9z6iAcaHRoT1P7r9zLt 
chrJw1sZ37wdrUHrTxiizUdZIe84vq4IdV1URK9z6iAcaHRoT1P7r9zLt thickBlackBorder scrollBar 
U3Q2S3tLGWkx thickBlackBorder scrollBar 
U3Q2S3tLGWkx thickBlackBorder scrollBar scrollBar 
U3Q2S3tLGWkx thickBlackBorder scrollBar scrollBar 
U3Q2S3tLGWkx thickBlackBorder scrollBar scrollBar 
